### ISSUE TITLE
Not compatible with python 3.7

### ISSUE THREAD

**Issue by maxsch3** [pawamoy/mkdocstrings#21]
Python 3.7.5

mkdocs.yml:
```
site_name: Keras batchflow
repo_url: https://github.com/maxsch3/keras-batchflow/
theme: readthedocs

plugins:
- mkdocstrings

nav:
  - Home: index.md
  - User guide: user_guide.md
  - Examples: examples.md
```

mkdocs serve
```
  File "/home/max/Code/opensource/batchflow/venv/lib/python3.7/site-packages/mkdocs/config/config_options.py", line 580, in load_plugin
    Plugin = self.installed_plugins[name].load()
  File "/home/max/Code/opensource/batchflow/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/home/max/Code/opensource/batchflow/venv/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/max/Code/opensource/batchflow/venv/lib/python3.7/site-packages/mkdocstrings/__init__.py", line 10, in <module>
    from .plugin import MkdocstringsPlugin
  File "/home/max/Code/opensource/batchflow/venv/lib/python3.7/site-packages/mkdocstrings/plugin.py", line 6, in <module>
    from .documenter import Documenter
  File "/home/max/Code/opensource/batchflow/venv/lib/python3.7/site-packages/mkdocstrings/documenter.py", line 12, in <module>
    from typing import Any, Callable, Dict, GenericMeta, List, Optional, Pattern, Tuple, Type, Union
ImportError: cannot import name 'GenericMeta' from 'typing' (/usr/lib/python3.7/typing.py)
```

Looks like GenericMeta has been removed from python specification https://www.python.org/dev/peps/pep-0560/

I found a [link](https://github.com/hsolbrig/PyShEx/issues/17) where people suggest the following fix:
```
try:
    from typing import GenericMeta  # python 3.6
except ImportError:
    # in 3.7, genericmeta doesn't exist but we don't need it
    class GenericMeta(type): pass
```

**Comment by pawamoy** [comment]
Yep, already fixed thanks to #2, I'll release it soon

**Comment by pawamoy** [comment]
Just released version 0.6.0. Don't hesitate to open new issues for things that do not work!
